### PR TITLE
feat: Apply adjusts in `HeroSection` and `BannerText` section

### PR DESCRIPTION
**Pull request by ArthurTriis1**
## What's the purpose of this pull request?

Apply adjusts in Hero Section to connect with CMS

### CODE REVIEW

File: packages/core/src/components/ui/BannerText/BannerText.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+   * The content for the h2 tag.
+   */
+  title: string
+  /**
+   * The content for the p tag below the h2.
+   */
+  caption: string
+}
+
+// TODO: Change actionPath and actionLabel with Link
```
Comment (hellofanny) [inline packages/core/src/components/ui/BannerText/BannerText.tsx:35]: I guess this is already done, right?

Comment (ArthurTriis1) [reply packages/core/src/components/ui/BannerText/BannerText.tsx:35]: yes!

Comment (ArthurTriis1) [reply packages/core/src/components/ui/BannerText/BannerText.tsx:35]:
```suggestion
```

File: packages/core/src/components/ui/BannerText/BannerText.stories.mdx
Comment (eduardoformiga) [inline packages/core/src/components/ui/BannerText/BannerText.stories.mdx:1]: should we remove this file already?

**Review by eduardoformiga — APPROVED** [review]
Good Job! 👏

File: packages/core/src/components/ui/BannerText/BannerText.stories.mdx
Comment (ArthurTriis1) [reply packages/core/src/components/ui/BannerText/BannerText.stories.mdx:1]: I think could be better remove when we move the BannerText to @faststore/components

**Review by hellofanny — APPROVED** [review]

**Pull request merged**